### PR TITLE
Split taxpayer info and refund info

### LINKED ISSUE #34: Replace Wages (Form W-2) text on the top of the page with different text depending on section or remove entirely

### LINKED ISSUE #43: rename family information page
Given the content of the family information page, the name should be changed to potentially refund information or contact & bank information, something along those lines.

### PR DESCRIPTION
**Pull request by zakpatterson**
Split taxpayer info and refund info, to prepare for expanding taxpayer with spouse and dependents

Fixes #34 
Fixes #43 

Along the way I also did some things I wish were separate PRs:

Rename a few files. Files with single exports of ReactElements are capitalized, files with multiple exports, like packages, are lowercased.

Pull out Save and Continue into reusable components.

### CODE REVIEW

**Comment by zakpatterson** [comment]
I'll rebase this after #48

**Comment by thegrims** [comment]
Since the W2 title text is being removed, should add 2em margin to the top of the form

**Comment by thegrims** [comment]
Could align w2 information text information left like other text in the form like so

```
<Box display="flex" justifyContent="flex-start">
          <strong>Input data from W-2</strong>
</Box>
```

File: src/components/paging.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement } from 'react'
+import { useHistory } from 'react-router-dom'
```
Comment (thegrims) [inline src/components/paging.tsx:2]: This looks like a different paging implementation than https://github.com/thegrims/UsTaxes/pull/50

Comment (zakpatterson) [reply src/components/paging.tsx:2]: Yep I split off #50 to work on this separately, plan on rebasing here after.

**Comment by thegrims** [comment]
The other two are fixed but it looks like w2jobinfo is still center aligned. Also, taxpayer information should probably be renamed to contact information

**Comment by zakpatterson** [comment]
> The other two are fixed but it looks like w2jobinfo is still center aligned. :heavy_check_mark: 

> Also, taxpayer information should probably be renamed to contact information
Agree as to current content, but spouse and dependent info is about to land on this page.

**Comment by thegrims** [comment]
Gotcha, that makes sense

**Comment by thegrims** [comment]
Should Refund Instructions be Refund Information?

**Comment by zakpatterson** [comment]
> Should Refund Instructions be Refund Information?

I don't know? Maybe at this point, but lines 34-36 of the 1040 can instruct the IRS to send the refund by check (no account number provided), by direct deposit, or by applying it to estimated tax payments for the succeeding year. 

So all of the pages are only partially baked at this point, should the titles and descriptions etc match the features they'll eventually have?

**Review by thegrims — APPROVED** [review]
LGTM

**Pull request merged**